app/controllers/student_learning_resources_controller.rb

```diff
@@ -1,5 +1,6 @@
 class StudentLearningResourcesController < ApplicationController
   before_action :authenticate_user!, :set_user
+  before_action :set_student_learning_resource, only: [:destroy]
```

Comment:
I don't see the need for a one-line before action only used for one single method

Reply:
I supposed it was a god practice.